src/ocean/meta/codegen/CTFE.d

```diff
@@ -51,15 +51,26 @@ unittest
     static assert (toString(14L) == "14");
     static assert (toString(long.max) == "9223372036854775807");
     static assert (toString(long.min) == "-9223372036854775808");
+    static assert (toString(int.min) == "-2147483648");
+    static assert (toString(int.max) == "2147483647");
+    static assert (toString(short.min) == "-32768");
+    static assert (toString(short.max) == "32767");
```

Comment:
For the sake of completeness check `ulong.max`, too. Values greater than `long.max` used to be a notorious source of trouble, and it's good to see they are supported now.